.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Bug Report
+about: Report a bug to help improve Chorus
+title: '[BUG] '
+labels: bug
+assignees: ''
+
+---
+
+## Bug Report
+
+### Description
+A clear and concise description of the bug.
+
+### Expected Behavior
+What you expected to happen.
+
+### Actual Behavior
+What actually happened.
+
+### Steps to Reproduce
+1. [Step 1]
+2. [Step 2]
+3. [Step 3]
+
+### Environment
+- **Chorus Version**: [e.g., v0.5.5]
+- **S3 Provider**: [e.g., AWS S3, Ceph RGW, MinIO]
+- **Chorus Setup**: [e.g.,K8s, Docker Compose, standalone binary]
+- **Other Details**: [e.g., Redis version, network configuration]
+
```

Comment:
Also would make sense to request chorus configuration and runtime info (os, distro, docker/not docker, etc.)

Reply:
we don't have os-specific code, but we ask if it is k8s/docker/binary deployment on line 29. i think it should be sufficient.